### PR TITLE
fix translations: removes Blood_IV "keep cold"

### PR DESCRIPTION
**Pull request by Elgin675**
**When merged this pull request will:**
- Because of this simple mention, many urban myths and legends continue to exist about blood conservation.
So I think it is best to remove it, especially because it can only create confusion.

### CODE REVIEW

File: addons/medical_treatment/stringtable.xml [hunk 1]
```diff
@@ -1473,20 +1473,20 @@
...
-            <Chinesesimp>血液, 用于补充伤者流失的血液 (需冷藏)</Chinesesimp>
-            <Chinese>血液, 用於補充傷者流失的血液 (需冷藏)</Chinese>
+            <English>Blood IV, for restoring a patients blood</English>
+            <Russian>Пакет крови для возмещения объёма потерянной крови</Russian>
+            <Spanish>Sangre intravenosa, para restarurar el volumen sanguíneo</Spanish>
+            <French>Culot sanguin pour transfustion IV, restaure le sang d'un patient.</French>
+            <Polish>Krew IV, używana do uzupełnienia krwi u pacjenta.</Polish>
+            <Hungarian>Vér-infúzió, intravénás bejuttatásra egy páciensnek</Hungarian>
+            <Italian>Sangue usato per ripristinare pazienti in cui si è verificata una perdita di sangue</Italian>
+            <German>Blut IV, Bluthaushalt des Patienten wiederherstellen.</German>
```
Comment (bux) [inline addons/medical_treatment/stringtable.xml]:
```suggestion
            <German>Blut IV, um den Bluthaushalt des Patienten wiederherzustellen.</German>
```

**Comment by jonpas** [comment]
> many urban myths and legends continue to exist about blood conservation.

:ok_hand: :joy:

**Comment by Italiandrache** [comment]
As implemented right now, is there any difference between Blood and Saline?

**Comment by Elgin675** [comment]
Absolutely no difference, no :).

**Pull request merged**